scopes/semantics/schema/mock/button-schemas.json

```diff
@@ -3411,45 +3435,8 @@
         "line": 1,
         "character": 1
       },
-      "exports": [
```

Comment:
Don't we loose tons of valuable info here? I guess this is the file we use for the tests and it shows 3,511 deletion